lib/src/main/java/com/alphawallet/token/entity/AttributeType.java

```diff
@@ -67,6 +69,9 @@ public AttributeType(Element attr, TokenDefinition def)
                 case "1.3.6.1.4.1.1466.115.121.1.27":
```

Comment:
Could we define these as constants to make them less error prone and simple?

Reply:
those are the ISO values which are used in the tokenscript. Are you proposing something like an enum abstraction? If so it's not a bad idea.

Reply:
Done